doc/api/unicode.md

```diff
@@ -0,0 +1,55 @@
+# icu
+
+    Stability: 1 - Experimental
```

Comment:
Since c809b8834541bf63b1a43d5a12833d98ffba834f we’re using blockquotes here (`> Stability: 1 - Experimental`) instead of indentation